client/src/utils/urlify.config.json

```diff
@@ -175,7 +175,7 @@
     ]
   },
   {
-    "RUSSIAN_MAP": [
+    "RUSSIAN_MAP:ru": [
```

Comment:
["х", "h"], -> ["х", "x"] or  ["х", "kh"]
["Х", "H"], -> ["Х", "X"] or  ["X", "Kh"]
["Щ", "Sh"], -> ["Щ", "Shh"]
["щ", "sh"], -> ["щ", "shh"]

see here: https://ifap.ru/library/gost/70342014.pdf (page 7 in pdf)

I would also reconsider transliteration for the Belarusian language, if possible

Reply:
@ACK1D I assume this would be a separate language code?

Is this correct?

```json
  # after Ukrainian
  {
    "BELARUSIAN_MAP:bu": [
      ["X", "Kh"],
      ["х", "kh"],
      ["Х", "X"],
      ["х", "x"],
      ["щ", "shh"],
      ["Щ", "Shh"]
    ]
  },
```

Note: We probably would add this as a separate PR, but just want to get the config right.

Reply:
@lb- no, what I sent is the incorrect current implementation for Russian according to Russian GOSTs. But, if we have disturbed this hive with the difference between Russian and Ukrainian, then it makes sense to make corrections for the Belarusian language as well.  

`{
    "RUSSIAN_MAP:ru": [
      ["а", "a"],
      ["б", "b"],
      ["в", "v"],
      ["г", "g"],
      ["д", "d"],
      ["е", "e"],
      ["ё", "yo"],
      ["ж", "zh"],
      ["з", "z"],
      ["и", "i"],
      ["й", "j"],
      ["к", "k"],
      ["л", "l"],
      ["м", "m"],
      ["н", "n"],
      ["о", "o"],
      ["п", "p"],
      ["р", "r"],
      ["с", "s"],
      ["т", "t"],
      ["у", "u"],
      ["ф", "f"],
      ["х", "x"], // current: "х", "h" but aslo allowed: "х", "kh"
      ["ц", "c"],
      ["ч", "ch"],
      ["ш", "sh"],
      ["щ", "shh"], // current: "щ", "sh"
      ["ъ", ""],
      ["ы", "y"],
      ["ь", ""],
      ["э", "e"],
      ["ю", "yu"],
      ["я", "ya"],
      ["А", "A"],
      ["Б", "B"],
      ["В", "V"],
      ["Г", "G"],
      ["Д", "D"],
      ["Е", "E"],
      ["Ё", "Yo"],
      ["Ж", "Zh"],
      ["З", "Z"],
      ["И", "I"],
      ["Й", "J"],
      ["К", "K"],
      ["Л", "L"],
      ["М", "M"],
      ["Н", "N"],
      ["О", "O"],
      ["П", "P"],
      ["Р", "R"],
      ["С", "S"],
      ["Т", "T"],
      ["У", "U"],
      ["Ф", "F"],
      ["Х", "X"], // current: "Х", "H" (changed to X, but aslo allowed: "Х", "Kh")
      ["Ц", "C"],
      ["Ч", "Ch"],
      ["Ш", "Sh"],
      ["Щ", "Shh"], // current: "Щ", "Sh" (changed to Shh)
      ["Ъ", ""],
      ["Ы", "Y"],
      ["Ь", ""],
      ["Э", "E"],
      ["Ю", "Yu"],
      ["Я", "Ya"]
    ]
  },`

Reply:
Ahh I see. I had a feeling that maybe the Russian ones were incorrect, it's from a port of the Django mapping. It's probably incorrect there.

I'll take a look, it might be best to do as you suggested and fix up these also in the same PR. 

Out of curiosity, because that PDF is a bit hard to parse for me. 

Does this site reflect the desired behaviour?
https://translit.cc/

Or do you know of something equivalent to this website?
https://czo.gov.ua/en/translit

Or is that PDF the best source of truth. I'll probably add these links to the JS code also.

Reply:
if you are select GOST 7.79 B in https://translit.cc/ then my proposed changes are relevant and correct. You can see it here: https://en.wikipedia.org/wiki/GOST_7.79-2000 

I don't know of any https://czo.gov.ua/en/translit analogs, but I'll ask around.

Reply:
if we talk about the Belarusian “default”:
```
{
    "BELARUSIAN_MAP:be": [
      ["а", "a"],
      ["б", "b"],
      ["в", "v"],
      ["г", "g"],
      ["д", "d"],
      ["е", "je"], // different to russian
      ["ё", "jo"], // different to russian
      ["ж", "zh"],
      ["з", "z"],
      ["і", "i"], // different to russian letter
      ["й", "j"],
      ["к", "k"],
      ["л", "l"],
      ["м", "m"],
      ["н", "n"],
      ["о", "o"],
      ["п", "p"],
      ["р", "r"],
      ["с", "s"],
      ["т", "t"],
      ["у", "u"],
      ["ў", "w"], // new letter
      ["ф", "f"],
      ["х", "h"], // if we are change ru to ["х", "x"] or ["х", "kh"]
      ["ц", "c"],
      ["ч", "ch"],
      ["ш", "sh"],
      ["ы", "y"],
      ["ь", ""],
      ["э", "e"],
      ["ю", "ju"], // different to russian
      ["я", "ja"], // different to russian
      ["А", "A"],
      ["Б", "B"],
      ["В", "V"],
      ["Г", "G"],
      ["Д", "D"],
      ["Е", "Je"], // different to russian
      ["Ё", "Jo"], // different to russian
      ["Ж", "Zh"],
      ["З", "Z"],
      ["І", "I"], // different to russian letter
      ["Й", "J"],
      ["К", "K"],
      ["Л", "L"],
      ["М", "M"],
      ["Н", "N"],
      ["О", "O"],
      ["П", "P"],
      ["Р", "R"],
      ["С", "S"],
      ["Т", "T"],
      ["У", "U"],
      ["Ў", "W"], // new letter
      ["Ф", "F"],
      ["Х", "H"], // if we are change ru to ["Х", "X"] or ["Х", "Kh"]
      ["Ц", "C"],
      ["Ч", "Ch"],
      ["Ш", "Sh"],
      ["Ъ", ""],
      ["Ы", "Y"],
      ["Ь", ""],
      ["Э", "E"],
      ["Ю", "Ju"], // different to russian
      ["Я", "Ja"] // different to russian
    ]
  },
  ```